packages/lesswrong/components/recommendations/ConfigurableRecommendationsList.tsx

```diff
@@ -5,7 +5,7 @@ import { Link } from '../../lib/reactRouterWrapper'
 import { getRecommendationSettings } from './RecommendationsAlgorithmPicker'
 import { forumTypeSetting } from '../../lib/instanceSettings';
 
-const recommendedName = forumTypeSetting.get() === 'EAForum' ? 'Forum Favorites' : 'Recommended'
+const recommendedName = forumTypeSetting.get() === 'EAForum' ? 'Forum Favorites' : 'Archive Recommendations'
```

Comment:
I assume that this is and less wrongs canonical name for this. I wonder if it's worth obstructing this into its own function e.g. `getRecommendationSectionName()`